packages/foreman/foreman-fapolicyd/foreman-fapolicyd.spec

```diff
@@ -0,0 +1,89 @@
+Name:     foreman-fapolicyd
+Version:  1.0.0
+Release:  1%{?dist}
+Summary:  Foreman fapolicyd rules
+
+Group:    System Environment/Base
+License:  GPLv3+
+URL:      https://theforeman.org
+Source0:  https://codeload.github.com/theforeman/%{name}/tar.gz/v%{version}#/%{name}-%{version}.tar.gz
+
+BuildArch: noarch
+
+Requires: fapolicyd
+Requires: foreman-proxy-fapolicyd
+
+%description
+Foreman fapolicyd rules
+
+%prep
+%setup -q -n %{name}-%{version}
+
+%build
+
+%install
+
+mkdir -p %{buildroot}%{_sysconfdir}/fapolicyd/rules.d
+cp 60-foreman.rules %{buildroot}%{_sysconfdir}/fapolicyd/rules.d/60-foreman.rules
+cp 61-foreman-proxy.rules %{buildroot}%{_sysconfdir}/fapolicyd/rules.d/61-foreman-proxy.rules
+
+%post
+
+if %{_bindir}/systemctl is-active --quiet fapolicyd.service; then
+  %{_sbindir}/fagenrules --load
+  # loading fagenrules leaves fapolicyd.service in a dead state, restart to ensure
+  %{_bindir}/systemctl restart fapolicyd.service
+else
+  %{_sbindir}/fagenrules --load
+fi
+
+%postun
+
+if %{_bindir}/systemctl is-active --quiet fapolicyd.service; then
+  %{_sbindir}/fagenrules --load
+  # loading fagenrules leaves fapolicyd.service in a dead state, restart to ensure
+  %{_bindir}/systemctl restart fapolicyd.service
+else
+  %{_sbindir}/fagenrules --load
+fi
+
+%files
+%defattr(-,root,root)
+%attr(0644,root,fapolicyd) %{_sysconfdir}/fapolicyd/rules.d/60-foreman.rules
+
+%package -n foreman-proxy-fapolicyd
+Summary: Foreman Proxy fapolicyd rules
+Group:   System Environment/Base
+
+Requires: fapolicyd
+
+%description -n foreman-proxy-fapolicyd
+Foreman Proxy fapolicyd rules
+
+%post -n foreman-proxy-fapolicyd
+
+if %{_bindir}/systemctl is-active --quiet fapolicyd.service; then
+  %{_sbindir}/fagenrules --load
+  # loading fagenrules leaves fapolicyd.service in a dead state, restart to ensure
+  %{_bindir}/systemctl restart fapolicyd.service
+else
+  %{_sbindir}/fagenrules --load
+fi
+
+%postun -n foreman-proxy-fapolicyd
+
+if %{_bindir}/systemctl is-active --quiet fapolicyd.service; then
+  %{_sbindir}/fagenrules --load
+  # loading fagenrules leaves fapolicyd.service in a dead state, restart to ensure
```

Comment:
I think this was fixed in `fapolicyd-1.1.3-8` via [BZ#2070639](https://bugzilla.redhat.com/show_bug.cgi?id=2070639): 
```console
[root@centos8-stream ~]# systemctl status fapolicyd
● fapolicyd.service - File Access Policy Daemon
   Loaded: loaded (/usr/lib/systemd/system/fapolicyd.service; disabled; vendor preset: disabled)
   Active: active (running) since Fri 2023-10-13 07:37:27 UTC; 1s ago
     Docs: man:fapolicyd(8)
  Process: 16522 ExecStart=/usr/sbin/fapolicyd (code=exited, status=0/SUCCESS)
  Process: 16497 ExecStartPre=/usr/sbin/fagenrules (code=exited, status=0/SUCCESS)
 Main PID: 16523 (fapolicyd)
    Tasks: 4 (limit: 37768)
   Memory: 43.3M
   CGroup: /system.slice/fapolicyd.service
           └─16523 /usr/sbin/fapolicyd

Oct 13 07:37:27 centos8-stream.tanso.example.com fapolicyd[16522]: Can handle 262144 file descriptors
Oct 13 07:37:27 centos8-stream.tanso.example.com fapolicyd[16523]: Initializing the trust database
Oct 13 07:37:27 centos8-stream.tanso.example.com systemd[1]: Started File Access Policy Daemon.
Oct 13 07:37:27 centos8-stream.tanso.example.com fapolicyd[16523]: Trust database migration will be performed.
Oct 13 07:37:27 centos8-stream.tanso.example.com fapolicyd[16523]: fapolicyd integrity is 0
Oct 13 07:37:27 centos8-stream.tanso.example.com fapolicyd[16523]: Loading rpmdb backend
Oct 13 07:37:27 centos8-stream.tanso.example.com fapolicyd[16523]: Creating trust database
Oct 13 07:37:27 centos8-stream.tanso.example.com fapolicyd[16523]: Loading trust data from rpmdb backend
Oct 13 07:37:27 centos8-stream.tanso.example.com fapolicyd[16523]: Loading trust data from file backend
Oct 13 07:37:27 centos8-stream.tanso.example.com fapolicyd[16523]: Starting to listen for events

[root@centos8-stream ~]# vi /etc/fapolicyd/rules.d/60-foreman.rules

[root@centos8-stream ~]# fagenrules --load

[root@centos8-stream ~]# systemctl status fapolicyd
● fapolicyd.service - File Access Policy Daemon
   Loaded: loaded (/usr/lib/systemd/system/fapolicyd.service; disabled; vendor preset: disabled)
   Active: active (running) since Fri 2023-10-13 07:37:27 UTC; 12s ago
     Docs: man:fapolicyd(8)
  Process: 16522 ExecStart=/usr/sbin/fapolicyd (code=exited, status=0/SUCCESS)
  Process: 16497 ExecStartPre=/usr/sbin/fagenrules (code=exited, status=0/SUCCESS)
 Main PID: 16523 (fapolicyd)
    Tasks: 4 (limit: 37768)
   Memory: 95.9M
   CGroup: /system.slice/fapolicyd.service
           └─16523 /usr/sbin/fapolicyd

Oct 13 07:37:27 centos8-stream.tanso.example.com fapolicyd[16523]: Loading trust data from rpmdb backend
Oct 13 07:37:27 centos8-stream.tanso.example.com fapolicyd[16523]: Loading trust data from file backend
Oct 13 07:37:27 centos8-stream.tanso.example.com fapolicyd[16523]: Starting to listen for events
Oct 13 07:37:36 centos8-stream.tanso.example.com fapolicyd[16523]: It looks like there was an update of the system... Syncing DB.
Oct 13 07:37:36 centos8-stream.tanso.example.com fapolicyd[16523]: Loading rpmdb backend
Oct 13 07:37:36 centos8-stream.tanso.example.com fapolicyd[16523]: Updating trust database
Oct 13 07:37:36 centos8-stream.tanso.example.com fapolicyd[16523]: Creating trust database
Oct 13 07:37:36 centos8-stream.tanso.example.com fapolicyd[16523]: Loading trust data from rpmdb backend
Oct 13 07:37:37 centos8-stream.tanso.example.com fapolicyd[16523]: Loading trust data from file backend
Oct 13 07:37:37 centos8-stream.tanso.example.com fapolicyd[16523]: Updated
```

Reply:
Tried to remove the systemd bits and install the policy, seems to work just fine once https://github.com/theforeman/foreman-fapolicyd/pull/3 is fixed, otherwise it *really* crashes `fapolicyd` but then even the restart doesn't help.